generator/04-syndesis-atlasmap.yml.mustache

```diff
@@ -185,14 +185,20 @@
             - --upstream=http://127.0.0.1:8080
             - --tls-cert=/etc/tls/private/tls.crt
             - --tls-key=/etc/tls/private/tls.key
-            - --cookie-secret=${OAUTH_COOKIE_SECRET}
+            - --cookie-secret=$(OAUTH_COOKIE_SECRET)
```

Comment:
Cool, didn't knew until now that k8s resolves `$(..)` as env vars for command args ....